src/app/screens/LNURLPay/index.tsx

```diff
@@ -418,6 +418,7 @@ function LNURLPay() {
                               value={valueSat}
                               onChange={(e) => setValueSat(e.target.value)}
                               fiatValue={fiatValue}
+                              hint={`Balance: ` + userBalance + ` sats`}
```

Comment:
@rolznz What's the best way to make this account balance & currency available to multiple places? 

Since we now have USD / EUR accounts (kollider) and moved the balance out of the dropdown we will now need this in several places in our application. 

What's the right way to solve this? Should we create a separate hook for that?

Reply:
I just found out about `balancesDecorated` which exactly does what I need :heavy_check_mark: